library/src/main/java/com/cube/storm/ui/model/page/ListPage.java

```diff
@@ -19,12 +21,57 @@
  * @project LightningUi
  */
 @NoArgsConstructor @AllArgsConstructor
-@Accessors(chain = true) @Data @EqualsAndHashCode(callSuper=false)
+@Accessors(chain = true) @Data @EqualsAndHashCode(callSuper = false)
 public class ListPage extends Page
 {
 	public static String CLASS_NAME = "ListPage";
+	public static final String ENGLISH_LANGUAGE_CODE = "en";
 
-	{ this.className = CLASS_NAME; }
+	{
+		this.className = CLASS_NAME;
+	}
+
+	/**
+	 * The array list of audios {@link VideoProperty}
+	 */
+	protected Collection<VideoProperty> audio;
+
+	/**
+	 * Returns the audio file for the user's current locale.
+	 * In case storm has audios and the current locale is not supported,
+	 * it will send the english audio by default.
+	 * The storm locale always have the format "usa_en" so
+	 * it will split the string to check the language
+	 *
+	 * @return The audio file if it exists
+	 */
+	public VideoProperty getCurrentLanguageAudioFile()
+	{
+		if (audio == null)
+		{
+			return null;
+		}
+
+		VideoProperty videoProperty;
+		VideoProperty auxVideoProperty = null;
+
+		while (audio.iterator().hasNext())
+		{
+			videoProperty = audio.iterator().next();
+			String[] localeArray = videoProperty.getLocale().split("_");
+
+			if (Locale.getDefault().getLanguage().equals(new Locale(localeArray[1]).getLanguage()))
```

Comment:
In classic Java fashion, d we want to ensure `localeArray` has at least 2 elements?

Reply:
Yes, thanks Ben :)